Controller/Payment/Webhook.php

```diff
@@ -265,7 +266,7 @@ protected function authorize(array $post)
         }
         catch (\Exception $e)
         {
-            $this->logger->info("Razorpay Webhook payment.authorized exeption, orderID:" . $orderId
+            $this->logger->critical("Razorpay Webhook payment.authorized exeption, orderID:" . $orderId
                                     ." and entity_id:".$salesOrder['entity_id']
```

Comment:
@enishant , what if sales order fetch don't give any result. This is not available `$salesOrder['entity_id']`.
Can you please correct all of these.

Reply:
@ramth05 Fixed exception log variables